test/smoke/appServers/jre.master.txt

```diff
@@ -1,2 +1 @@
-openjdk:7-jre
 openjdk:8-jre
```

Comment:
@littleaj is there way to disable JRE-7 for Springboot smoke test? The async methods which return `CompletableFuture` does not work with JRE-7.

Reply:
Not yet. I'll work on it

Reply:
It's not immediately needed. Just keep in your backlog.